kubernetes/token.go

```diff
@@ -1,16 +1,23 @@
 package kubernetes
 
-import "io/ioutil"
+import (
+	"io/ioutil"
+	"time"
+
+	kialiConfig "github.com/kiali/kiali/config"
+)
 
 // Be careful with how you use this token. This is the Kiali Service Account token, not the user token.
 // We need the Service Account token to access third-party in-cluster services (e.g. Grafana).
 
-const DefaultServiceAccountPath = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+var DefaultServiceAccountPath = "/var/run/secrets/kubernetes.io/serviceaccount/token"
 
 var KialiToken string
+var tokenRead time.Time
+var tokenExpireDuration time.Duration
```

Comment:
Do we need to add more global variables?

Reply:
tokenRead would be needed to keep track of when the token was read in the next time. 
Probably tokenExpireDuration doesn't need to be global. Just to avoid read kConfig again.